README.md

```diff
@@ -29,6 +29,9 @@
   - While an unbalanced tree could increase the search time from logarithmic to linear, the change from 6.65 to 100 seconds makes me think that other factors are also envolved
   - For example, if there were multiple programs running simultaneously, or you are running this search on a different system entirely
   - The combination of an unbalanced tree and the senario which it is being run in could explain such a large difference in runtime
+  - Some additional reasons that could contribute to this large search time could be the implementation of the binary search tree, this time focusing on things like extra comparisions or
+    excessive logging of each step of the search could contribute to the change in runtime
+  - If a recurssive approach is used. Inefficient recursion can lead to excessive recursive calls which can cause an increase in memory usage and processing time an iterative approach would prevent this from happening
```

Comment:
Not sure about these last two reasons -- can you maybe give examples to make them more concrete please?

Reply:
So I had added another possible cause of the extended runtime. I think there could be cases for the two that you commented on however for this case it would not seem logical since we have already run the program with 1,000 elements and did not run into any of those situations.

Reply:
These are still not really specific enough. There also seems to be some overlap between the reasons.